src/Conan.cmake

```diff
@@ -77,6 +77,23 @@ macro(run_conan)
         # CONAN_ENV should be redundant, since the profile can set CC & CXX
       endif()
 
+      if("${ProjectOptions_CONAN_PROFILE}" STREQUAL "")
+        set(CONAN_DEFAULT_PROFILE "default")
+      else()
+        set(CONAN_DEFAULT_PROFILE ${ProjectOptions_CONAN_PROFILE})
+      endif()
```

Comment:
This can be moved under the `if` section of the next condition. It would potentially simplify the code a bit.